app/catalog_data.go

```diff
@@ -16,8 +21,71 @@ const (
 	systemLibraryURL    = "https://git.rancher.io/system-charts"
 	systemLibraryBranch = "master"
 	systemLibraryName   = "system-library"
+
+	defSystemChartVer = "manage.cattle.io/default-system-chart-version"
+)
+
+var (
+	sysDefaultBranch = os.Getenv("SYSTEM_CHART_DEFAULT")
```

Comment:
need to get rid of this var completely. should need to have a global scoped var for ths